examples/README.md

```diff
@@ -0,0 +1,7 @@
+# saucectl espresso examples
+
+Here we have a list additional examples that cover more complicated setups.
+
+| Example | Description |
+| ------- | ----------- |
+| [cucumber](cucumber) | Showcases how to run tests using espresso with cucumber |
```

Comment:
what about: `Showcases a setUp to run tests using espresso with cucumber` ? We're not showing how to write the tests, or the setup in your project, is just an example with config and applications using cucumber

Reply:
Sound good to me! I renamed it.